packages/examples/basic/package.json

```
@@ -1,10 +1,13 @@
 {
   "private": "true",
   "scripts": {
-    "start": "react-scripts start"
+    "start": "DISABLE_ESLINT_PLUGIN=true react-scripts start",
+    "analyze": "source-map-explorer 'build/static/js/*.js' --gzip",
```

Should we make this run build everytime we run analyze? Without a build present, analyze will just fail
```suggestion
    "analyze": "yarn build ; yarn source-map-explorer 'build/static/js/*.js' --gzip",
```

---

This is build of the app and not build of blade. The build of app won't run on install and its a bit handy to have one command that runs build and analyze when we change something. Added one "build:app" command to analyze script